### PR TITLE
Fixes #25735 - improve the warning on bulk deletion

### PR DESCRIPTION
**Pull request by ares**
warn

@lhellebr any comments to the text?

### CODE REVIEW

**Review by lzap — DISMISSED** [review]
Maybe making *destroy* and *will not* bold?

**Review by chris1984 — DISMISSED** [review]
+1 to @lzap comment, looks good overall

**Comment by ares** [comment]
new versions with bold text

a
b

@lhellebr any comments to these strings? I also changed existing destroy -> delete even that was unrelated to the new warning.

**Comment by KamikazeCZ** [comment]
@ares Looks good!

File: app/views/hosts/multiple_destroy.html.erb [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+  <% if Setting[:destroy_vm_on_host_delete] %>
+    <% extra_warning = _("This <b>will delete</b> the linked VMs and their disks, and is irreversible.") %>
+  <% else %>
+    <% extra_warning = _("It is irreversible, but VMs and their disks <b>will not be deleted</b>.") %>
+  <% end %>
+
   <%= alert :class => 'alert-warning',
-  :text => _("This might take a while, as all hosts, facts and reports will be destroyed as well"),
-  :header => '' %>
+    :text => (_("This might take a while, as all hosts, facts and reports <b>will be deleted</b> as well. %s This behavior can be changed in global settings.") % extra_warning).html_safe,
```
Comment (tbrisker) [inline app/views/hosts/multiple_destroy.html.erb]: Perhaps add the specific setting name?

**Comment by lhellebr** [comment]
@ares Looks good!

**Comment by ares** [comment]
@tbrisker updated the text and also on single host delete to keep the message consistent, please take another look

**Review by tbrisker — APPROVED** [review]
Thanks @ares !

**Pull request merged**